mrp_food/models/product_product.py

```diff
@@ -34,6 +34,7 @@ class ProductProduct(models.Model):
 
```

Comment:
```suggestion
    # because the computation is based on mrp.bom.line,
    # that is not available for user that doesn't belong to mrp user group
```